cmd/controller/main.go

```diff
@@ -56,7 +56,7 @@ var (
 )
 
 // main starts the operator for the gameserver CRD
-func main() {
+func main() { // nolint: gocyclo
```

Comment:
Not for this PR - but we should make a ticket for this, because we should fix this.